### PR TITLE
fixes of obvious code mistakes

### PR DESCRIPTION
**Pull request by DamirAinullin**
fixes of some simple obvious code mistakes and warnings.

### CODE REVIEW

File: Nodejs/Product/Profiling/Profiling/StandaloneTargetView.cs [hunk 1]
```diff
@@ -152,7 +152,7 @@ public StandaloneTargetView(StandaloneTarget template)
                     WorkingDirectory.IndexOfAny(Path.GetInvalidPathChars()) == -1 &&
                     (Path.IsPathRooted(ScriptPath) || Path.IsPathRooted(WorkingDirectory)) &&
                     File.Exists(Path.Combine(WorkingDirectory, ScriptPath)) &&
-                    (string.IsNullOrEmpty(WorkingDirectory) || Directory.Exists(WorkingDirectory)) &&
+                    (WorkingDirectory == string.Empty || Directory.Exists(WorkingDirectory)) &&
```
Comment (mjbvz) [inline Nodejs/Product/Profiling/Profiling/StandaloneTargetView.cs]: This changes the behavior and some analyzers will complain about comparing to `string.Empty` directly. Is there a specific reason for this change?

**Comment by mjbvz** [comment]
Thanks for the fixes. Just one small comment to take a look at before we merge this in.

What tool did you use to find these?

File: Nodejs/Product/Profiling/Profiling/StandaloneTargetView.cs (hunk 1 above)
Comment (DamirAinullin) [inline Nodejs/Product/Profiling/Profiling/StandaloneTargetView.cs]: We use WorkingDirectory at the line 152, before check it for null. So I can suppose that check is not needed. Maybe NullReferenceException is possible  at the line 152. Maybe we should check it for null before using.

**Comment by DamirAinullin** [comment]
Thanks for positive response. I just try to help your project. I use static analyser PVS-Studio. It is a wonderful tool.

**Comment by mjbvz** [comment]
Thanks for details and thanks again for finding these. LGTM, merging this in now.

**Pull request merged**